README.md

```diff
@@ -556,8 +556,8 @@ A `git stash` is basically a stack of changes on which you store any changes to
 
 The commands you'll mostly use are `git stash` which places any modifications to the _Working Directory_ on the stash, and `git stash pop` which takes the latest change that was stashed and applies it the to the _Working Directory_ again. 
 
-Just like the stack commands it's named after, `git stash pop` removes the latest stashed change before applying it again. 
-If you want to keep the stashed changes, you can use `git stash apply`, which doesn't remove them from the stash before applying them. 
+Just like the stack commands it's named after, `git stash pop` removes the latest stashed change before applying these changes to the working directory. 
+If you want to keep the stashed changes, you can use `git stash apply`, this applies the latest stashed changes to the working directory without removing them from the stash being maintained. 
```

Comment:
```suggestion
If you want to keep the stashed changes, you can use `git stash apply`, this applies the latest stashed changes to the working directory without removing them from the stash. 
```

Personally I think the `being maintained` doesn't really help the sentence (your other additions do). 

What do you think?

Reply:
yes you are right there @UnseenWizzard, will edit that part.